### DISCUSSION TITLE
Handling of derived currencies such as GBP/GBp

### DISCUSSION THREAD

**Discussion by ups7** [ghostfolio/ghostfolio#2617] [Ideas]
Hello,

I'd like to suggest some improvements to the way derived currencies such as GBP and GBp are handled. At the moment, Ghostfolio treats them as "separate" currencies, which leads to some issues as I'll try to explain below:

1. Many views show the value in the trading currency which is often (but not always) GBp, such as activities list. But I will prefer that they showed the values in GBP as that's more understandable, I doubt anyone thinks in pennies. I already have GBP set as the default currency but it's only used for totals/summaries.
2. While trading is usually in GBp, different brokers use either GBP or GBp in their transaction history/CSV export which means I have to do manual work every time I want to add an activity. This might be helped if Ghostfolio accepted both GBP and GBp as valid currencies if the holding is traded in one of these currencies (as returned by Yahoo Finance).

Any thoughts?

**Comment by bats77** [comment]
Maybe just a way to code a kind of alias.
Same worry with ZAR and some stock markets and brokers that prefer to work on smallest cents or some others which are working with a 4 to 5 decimals.
